data/pam_test/pam.sh

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env bats
+# pam-test tool from the url https://github.com/pbrezina/pam-test
```

Comment:
please package the pam-test tool and add it to QA Head, instead of adding the binary.

Reply:
pam-test was added into qa-head: http://download.suse.de/ibs/QA:/Head/SLE-15-SP2/src/pam-test-0.0+git.20161214-qa.1.3.src.rpm